src/olympia/accounts/utils.py

```diff
@@ -11,12 +11,24 @@
 
 import boto3
 import six
-from six.moves.urllib_parse import urlencode
+from six.moves.urllib_parse import urlencode, urlparse
 
 from olympia.accounts.tasks import primary_email_change_event
 from olympia.core.logger import getLogger
 
 
+def _is_safe_url(url, request):
+    """Override the Django `is_safe_url()` to pass a configured list of allowed
+    hosts and enforce HTTPS."""
+    allowed_hosts = (
+        settings.DOMAIN,
+        urlparse(settings.CODE_MANAGER_URL).netloc,
+    )
```

Comment:
I wonder if this should be a setting.

There is already an `ALLOWED_HOSTS` setting but that's being used for something else entirely (e.g csrf protection).

So maybe `ALLOWED_LOGIN_HOSTS` which is being set to these values? But then again we'd have to overwrite it explicitly in every hosting environments setting then which is kinda nasty too.

Maybe it's fine like it is here, just thinking loud :thinking: